src/rules/selector-attribute-quotes/__tests__/index.js

```diff
@@ -0,0 +1,92 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+
+  accept: [ {
+    code: "[title] { }",
+  }, {
+    code: "a[target=\"_blank\"] { }",
+  }, {
+    code: "[class|=\"top\"] { }",
+  }, {
+    code: "[title~=\'text\'] { }",
+  }, {
+    code: "[data-attribute=\'component\'] { }",
+  } ],
+
+  reject: [ {
+    code: "a[title=flower] { }",
+    message: messages.expectedQuotes,
```

Comment:
We idiomatically name these `messages.expected` and `messages.rejected` for this type of rule. Can you rename them please?